phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/assoc/HpoAssociationParser.java

```diff
@@ -136,10 +136,9 @@ private void ingestPhenotypeHpoaFile() {
     if (! phenotypeDotHpoaFile.exists()) {
       throw new PhenolRuntimeException("Cannot find phenotype.hpoa file");
     }
-    List<DiseaseDatabase> desiredDatabasePrefixes=ImmutableList.of(OMIM);
-    Map<TermId, HpoDisease> diseaseMap = HpoDiseaseAnnotationParser.loadDiseaseMap(this.phenotypeDotHpoaFile.getAbsolutePath(),
-      hpoOntology,
-      desiredDatabasePrefixes);
+    Set<DiseaseDatabase> desiredDatabasePrefixes= new HashSet<>();
```

Comment:
In this case, the line `139` is not determinant whether to use `List` or `Set`. The `desiredDatabasePrefixes` is  used downstream in `HpoDiseaseAnnotationParser`. 

The `HpoDiseaseAnnotationParser` uses the database prefixes to filter out annotation lines, e.g. `ORPHA` lines if only `OMIM` entries are requested. The filtering uses `contains`, which is *O(1)* in sets, and *O(n)* in lists, I assume `Set` is better here. Also, I think there is no need for predictable iteration here, since the code only tests membership.

Reply:
Fair enough. `Set.of(OMIM)` would still be preferable to a two line assignment. This is using/moving to Java 11 now isn't it?

Reply:
No, it's still Java 8 :/.

I want to pass through the library to replace immutable Guava collections by Java collections.